I'm sorry, but an uncaught exception occurred.

While loading <renpy.display.im.Image object ('gui/button/cursor.ani') at 0x00000000056e0730>:
  File "renpy/common/00start.rpy", line 211, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 388, in script
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 388, in <module>
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 317, in _m1_00gltest__gl_test
    _gl_performance_test()
  File "renpy/common/00gltest.rpy", line 340, in _gl_performance_test
    ui.interact(suppress_underlay=True, suppress_overlay=True)
Exception: Could not load image 'gui/button/cursor.ani' (ani files are not supported by Ren'Py): error('Unsupported image format')

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 211, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 388, in script
    $ __gl_test()
  File "D:\fanbase_project\KTPVisualNovel\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\fanbase_project\KTPVisualNovel\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00gltest.rpy", line 388, in <module>
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 317, in _m1_00gltest__gl_test
    _gl_performance_test()
  File "renpy/common/00gltest.rpy", line 340, in _gl_performance_test
    ui.interact(suppress_underlay=True, suppress_overlay=True)
  File "D:\fanbase_project\KTPVisualNovel\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\fanbase_project\KTPVisualNovel\renpy\display\core.py", line 2122, in interact
    self.start()
  File "D:\fanbase_project\KTPVisualNovel\renpy\display\core.py", line 1036, in start
    surf = renpy.display.im.load_surface(fn)
  File "D:\fanbase_project\KTPVisualNovel\renpy\display\im.py", line 2055, in load_surface
    return cache.get(image(im))
  File "D:\fanbase_project\KTPVisualNovel\renpy\display\im.py", line 309, in get
    surf = image.load()
  File "D:\fanbase_project\KTPVisualNovel\renpy\display\im.py", line 761, in load
    raise e
  File "D:\fanbase_project\KTPVisualNovel\renpy\display\im.py", line 722, in load
    surf = renpy.display.pgrender.load_image(f, filename)
  File "D:\fanbase_project\KTPVisualNovel\renpy\display\pgrender.py", line 201, in load_image
    raise Exception("Could not load image {!r}{}: {!r}".format(filename, extra, e))
Exception: Could not load image 'gui/button/cursor.ani' (ani files are not supported by Ren'Py): error('Unsupported image format')

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.0.24012702
KTPVisualNovel 1.0
Sat Mar 23 14:42:55 2024
